modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -291,19 +291,19 @@ Besides the default flag `--help`, this command also accepts:
 > secrets-mgr:certificate:replace  [flags] 
 ----
 
-Replaces an existing certificate secret you pass in `--id`, from the secret group you pass in `--group-id`, using the type you pass in `--type`. 
+Replaces an existing certificate secret specified in `--id`, from the secret group specified in `--group-id`, using the type specified in `--type`. 
 
 Besides the default flag `--help`, this command also accepts:
 
 [%header,cols="30a,40a,30a"]
 |===
 |Flag | Description |  Example
-|`--id`| Mandatory. Secret ID. | `--id 030493`
-|`--group-id`|Mandatory. Secret Group ID. | `--group-id 000000`
+|`--id`| Mandatory. Secret ID. | `--id 6e8417f6-2ca7-417a-82b6-047189a18b53`
+|`--group-id`|Mandatory. Secret Group ID. | `--group-id 1fec0a49-1551-4199-bfcc-cf0352d0f29d`
 |`--type`| Mandatory. Certificate secret type. The value must match the existing secret type. | `--type PEM`
 |`--name` | New name for your shared secret | `--name TestSharedSecret`
-|`--cert-file` |Certificate file type | `--cert-file json`
-|`--expiration-date`| Expiration date for the secret. | `--expiration-date 01/01/2025`
+|`--cert-file` |Certificate file type | `--cert-file /example-cert.pem`
```

Comment:
Minor - The example file path must be  ./example-cert.pem.   (period is missing before the slash)